### ISSUE TITLE
How to disable md-tabs

### ISSUE THREAD

**Issue by nmindz** [jadjoubran/laravel5-angular-material-starter#93]
I've been looking forward how to disable a few `<md-tab>`'s from my `<md-tabs>` section with no luck. If I add a `disabled` attribute as the [Angular Material API suggests](https://material.angularjs.org/latest/api/directive/mdTab), the attribute itself seems gone in the _element inspector_ (**both** Safari and Chrome.) and if I add it manually (within the inspector.), the tab will be properly _styled_ as disabled but _still function as it normally would_ (so, **not effectively** disabled.).    

May anyone guide me in the right direction? Here's a code example of what is not working:    

``` html
<div ng-controller="EmpresaNewCtrl" ng-cloak class="Page-Container md-inline-form">
    <md-content>
        <md-tabs md-dynamic-height md-border-bottom md-center-tabs>
            <md-tab label="transportadora">
                <!-- BEGIN TAB TRANSPORTADORA -->
                <md-tabs md-dynamic-height md-border-bottom md-center-tabs>
                    <md-tab label="Dados Gerais" disabled>
                        <!-- BEGIN SUB-TAB DADOS GERAIS -->
                        <md-content class="md-padding md-padding-top-x2">
                            <form name="empresaForm">
                                <div layout layout-sm="column">
                                    <md-input-container flex>
                                        <label>Razão Social</label>
                                        <input ng-model="empresa.razaoSocial">
                                    </md-input-container>
                                    <md-input-container>
                                        <label>CNPJ</label>
                                        <input ng-model="empresa.doc1">
                                    </md-input-container>
                                </div>
                                <div layout layout-sm="column">
                                    <md-input-container flex>
                                        <label>Nome Fantasia</label>
                                        <input ng-model="empresa.nomeFantasia">
                                    </md-input-container>
                                    <!-- <md-datepicker ng-model="placeholder.model"></md-datepicker> -->
                                </div>
                                ...
```

And this is the browser output of the `<md-tab>` element:    

``` html
<md-tab-item tabindex="-1" class="md-tab ng-scope ng-isolate-scope md-active" style="max-width: 957px;" ng-repeat="tab in $mdTabsCtrl.tabs" role="tab" aria-controls="tab-content-5" aria-selected="true" aria-disabled="false" ng-click="$mdTabsCtrl.select(tab.getIndex())" ng-class="{                      'md-active':    tab.isActive(),                      'md-focused':   tab.hasFocus(),                      'md-disabled':  tab.scope.disabled                  }" ng-disabled="tab.scope.disabled" md-swipe-left="$mdTabsCtrl.nextPage()" md-swipe-right="$mdTabsCtrl.previousPage()" md-template="::tab.label" md-scope="::tab.parent"><span class="ng-scope">Dados Gerais</span><div class="md-ripple-container" style="background-color: rgba(0, 151, 167, 0.0980392);"><div class="md-ripple md-ripple-placed md-ripple-scaled md-ripple-visible" data-counter="2" style="background-color: rgb(0, 151, 167); border-color: rgb(0, 151, 167); width: 168.01217217808954px; height: 168.01217217808954px; margin-top: -84.00608608904477px; margin-left: -84.00608608904477px; left: 50%; top: 50%;"></div></div></md-tab-item>
```

In this case I actually have two `<md-tabs>` nested (so a top navigation tab and a context navigation tab.) and what I really needed was to **hide all** tabs _but the main_ (the one which would be selected by default.) from the second tab level (the context one.) when I'm not editing the form. Or just hide the second nested tab altogether unless I'm updating the model.    

Is there any way of doing this within the starter pack? If not, are there any suggestions to achieve the desired behavior?

**Comment by jadjoubran** [comment]
Hi @nmindz 

Thanks for opening an issue here!
Keep in mind that this issue is solely related to Angular Material and this starter package won't change the original behavior - but I'd love to answer your Angular Material questions as well so feel free to ask more :D

I just had a look at Angular Material's demo and you can do this by using the following attribute:

`ng-disabled="'true'"` instead of `disabled`

Make sure to copy paste it correctly: `double quote` `single quote` `true` `single quote` `double quote`
You can also pass in a model value here so it would be:

`ng-disabled="controller.hasInfo"`

Let me know if this works for you

Cheers!

**Comment by nmindz** [comment]
Hi, @jadjoubran !    

Yes I knew that it was solely related to Angular Material but since a few "recipes" have been added to the starter pack (and as I've seen, there are a few features that ng-material is still missing such as an `<md-table>` native directive.) I wondered if there was such a functionality or if it had been implemented in this starter pack.  
As you've read before I tried the usual method and even looked for a API reference but it is either outdated or I did not find the correct reference. I'll surely try the `ng-disabled` attribute and let you know. I'm still learning Angular (this is my first project with it... hehehe.).    

Thanks a lot bud.  
Cheers! :+1:
